dojo/tools/hcl_appscan/parser.py

```diff
@@ -24,60 +33,85 @@ def get_findings(self, file, test):
-                            severity = item.text
+                            output = self.xmltreehelper(item)
+                            severity = output.strip(" ").capitalize()
                         case 'cwe':
-                            cwe = item.text
+                            cwe = int(self.xmltreehelper(item))
                         case 'remediation':
-                            remediation = item.text
+                            remediation = self.xmltreehelper(item)
                         case 'advisory':
-                            advisory = item.text
+                            advisory = self.xmltreehelper(item)
+                        case 'issue-type':
+                            title = self.xmltreehelper(item)
+                            description = description + "Issue-Type:" + title + "\n"
                         case 'issue-type-name':
-                            issuetypename = item.text
-                            description = description + "Issue-Type-Name: " + issuetypename + "\n"
+                            title = self.xmltreehelper(item)
+                            description = description + "Issue-Type-Name:" + title + "\n"
                         case 'location':
-                            location = item.text
-                            description = description + "Location: " + location + "\n"
+                            location = self.xmltreehelper(item)
+                            description = description + "Location:" + location + "\n"
                         case 'domain':
-                            domain = item.text
-                            description = description + "Domain: " + domain + "\n"
+                            domain = self.xmltreehelper(item)
+                            title += "_" + domain
+                            description = description + "Domain:" + domain + "\n"
+                        case 'threat-class':
+                            threatclass = self.xmltreehelper(item)
+                            description = description + "Threat-Class:" + threatclass + "\n"
+                        case 'entity':
+                            entity = self.xmltreehelper(item)
+                            title += "_" + entity
+                            description = description + "Entity:" + entity + "\n"
+                        case 'security-risks':
+                            security_risks = self.xmltreehelper(item)
+                            description = description + "Security-Risks:" + security_risks + "\n"
+                        case 'cause-id':
+                            causeid = self.xmltreehelper(item)
+                            title += "_" + causeid
+                            description = description + "Cause-Id:" + causeid + "\n"
+                        case 'url-name':
+                            urlname = self.xmltreehelper(item)
+                            title += "_" + urlname
```

Comment:
Same comment as above

```suggestion
                            title += "_" + urlname.strip()
```